requirements/requirements.txt

```diff
@@ -2,6 +2,7 @@ click
 pyfastx
 torch
 pytorch-lightning<2.0.0 # temp - need to make all tests pass with this version first
+torchvision<0.15  # temp
```

Comment:
still required? add a comment that explains the issue so we will be able to revert.
Also create a trello card to resolve this.

Reply:
Nice catch. I'm checking it now :)